### PR TITLE
Fix onboarding rendering

### PR DESCRIPTION
**Pull request by jessfraz**
- ensures we are reusing the types we have on the rust side so we can parse onboarding status properly

closes https://github.com/KittyCAD/modeling-app/issues/4787

### CODE REVIEW

File: src/components/ModelingSidebar/ModelingSidebar.tsx [hunk 1]
```diff
@@ -41,7 +42,7 @@ export function ModelingSidebar({ paneOpacity }: ModelingSidebarProps) {
   const onboardingStatus = settings.context.app.onboardingStatus
   const { send, context } = useModelingContext()
   const pointerEventsCssClass =
-    onboardingStatus.current === 'camera' ||
+    onboardingStatus.current === onboardingPaths.CAMERA ||
```
Comment (nadr0) [inline src/components/ModelingSidebar/ModelingSidebar.tsx:45]: Is it safe to say that `onboardingStatus.current` will now be of the format `/camera`? 

I looked at the constant you have here 
```
CAMERA: '/camera',
```

but the old code says `'camera'`. Is this a bug or the new expected constant?

Comment (jessfraz) [reply src/components/ModelingSidebar/ModelingSidebar.tsx:45]: that was a bug

Comment (jessfraz) [reply src/components/ModelingSidebar/ModelingSidebar.tsx:45]: the types actually check now :) there was lots of loose holes here

Comment (nadr0) [reply src/components/ModelingSidebar/ModelingSidebar.tsx:45]: perfect

**Review by nadr0 — APPROVED** [review]
Was able to reproduce the issue locally. The new constants are correct for the pathing.

**Pull request merged**